### DISCUSSION TITLE
Having issues installing League of Legends on Linux

### DISCUSSION THREAD

**Discussion by AlvaroUE** [lutris/lutris#5331] [General]
Hi guys, I am having trouble with the installation of League of Legends on Linux Mint. I am installing it on Lutris. I have tried Wine ge-lol-8-27 and Wine 9.0, but no one works. 
It is a bit confusing because the two downloads on the launcher goes well, but when I close it and try to relaunch LoL, it does not work. 
I saw that, during the installation on the latest launcher (in the page of login), in the lutris consol I get an error: "0544:fixme:dxgi:d3d11_swapchain_Present1 Ignored present parameters 009AED70".

Could you please help me?

**Comment by strycore** [comment]
First of all, don't even try running LoL on anything else other than wine-ge-lol. It is made for LoL for a reason.
Second, LoL is going to not work on Linux in a few weeks so might as well start playing Dota now.
